### DISCUSSION TITLE
切换 umi 4 的 bundler

### DISCUSSION THREAD

**Discussion by jh8w8m** [umijs/umi#11035] [Q&A]
请问下有什么方法切换掉 umi 4 的 bundler 吗？
我自己写了个基于 webpack 的 bundler，想在我以前用 umi 写的项目中测试一下。比如现在是 max build 打包，我切换掉用另一种命令打包这样。

**Comment by fz6m** [comment]
bundler 是 umi 内的一个概念，目前只有 webpack （默认）和 `vite: {}` ，没法自己定义。
